web/components/input/text/TextInput/TextInput.sass

```diff
@@ -0,0 +1,57 @@
+div.text-input
+  margin: 5px
+
+  *
+    font-family: Apercu Pro
+    font-weight: normal
+    font-size: 18px
+
+  > label
+    /* Body text */
+    color: $primary
+
+    > input
+      background: $white
+      width: 274px
+      height: 40px
+      border: 1px solid $grey-slate
+      box-sizing: border-box
+      border-radius: 4px
+
+      padding-left: 16px
+      padding-right: 16px
+      padding-top: 8px
+      padding-bottom: 8px
+
+      margin-top: 8px
+
+      color: $grey-dark
+
+      &.error
+        border-color: $error
+
+      &:hover:disabled
+        cursor: not-allowed
+        border-color: $grey-slate
+
+      &::placeholder
```

Comment:
Should sort by CSS type (`&.foo`s first, then `&:bar`s, then `&::bob`s, whichever order is fine)